### ISSUE TITLE
Changed Keybindings, Still Overrides

### ISSUE THREAD

**Issue by redbassett** [maltize/sublime-text-2-ruby-tests#247]
Related to #238, I like using my Super+Shift+T command to reopen the most recently closed tab. I have re-mapped RubyTest to use Super+Shift+Y, and this key command works (and is listed in the RubyTest menu), however when in a test file, Super+Shift+T still runs tests rather than opening the last closed file.

**Comment by jamonholmgren** [comment]
@redbassett I had the same issue. Just add it again to your user key bindings file.
```json
  { "keys": ["super+shift+t"], "command": "reopen_last_file" },
```

**Comment by redbassett** [comment]
Glad to see re-adding it is an option (hadn't tried yet). That said, any idea how to stop it from overriding in the first place once the keybinding is changed?

**Comment by jamonholmgren** [comment]
No...once this workaround worked, I quit looking for a real solution. :)

**Comment by redbassett** [comment]
I'll keep this issue open in hopes of getting it resolved correctly, but the custom keybinding works perfectly in the mean time.

**Comment by sentience** [comment]
For the record, `"super+shift+r"` also conflicts with the default key binding for **Goto Symbol in Project…** in Sublime Text 3.

**Comment by gerardroche** [comment]
Add a preference for enable/disable the keymaps. I do this for enabling vi style keymaps in sublime-phpunit. See https://github.com/gerardroche/sublime-phpunit#key-bindings.
